extractors/tiffimagingextractor/tiffimagingextractor.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+from pathlib import Path
+from ..imagingextractor import ImagingExtractor
+from ..extraction_tools import ArrayType, PathType
+
+
+class TiffImagingExtractor(ImagingExtractor):
+    def __init__(self, filepath: PathType, sampling_frequency=None,
+                 channel_names=None):
+
+        ImagingExtractor.__init__(self)
+        self.filepath = Path(filepath)
+        self._sampling_frequency = sampling_frequency
+        assert self.filepath.suffix in ['.tiff', '.tif']
+        # TODO placeholder
+        self._video = np.load(self.filepath)
```

Comment:
this should be memmap-like

Reply:
just copy-pasted the numpy extractor here for now

Reply:
this memmaps automatically (it has a `mmap_mode` arg!)

Reply:
updated!